client/src/lib/store/track.ts

```diff
@@ -87,4 +104,28 @@ export const useTrack = create<TrackState>()((set) => ({
       }
       return { ...state };
     }),
-}));
\ No newline at end of file
+
+  updateInterviewingRound: (applicationId, round) =>
```

Comment:
Just have a general function that updates the entire metadata of an application instead of changing a single attribute of an application (in this case the interview round)

Reply:
Yes it has been fixed.